csc2/maccparse.y

```diff
@@ -146,9 +144,24 @@ cnstrtnamedstart: string T_EQ string '-' T_GT {
                       set_constraint_name($1);
                   }
 
+/* Named check constraint */
+cnstrtnamedcheck: T_CHECK string T_EQ string {
```

Comment:
support double quote like index expr (anything between {} gets shipped to SQLite wholesale)